python/lsst/ts/watcher/rules/cp_verify_alarm.py

```diff
@@ -0,0 +1,266 @@
+    ``config.calibration_type`` and ``config.ocps_index``, respectively.
+    """
+
+    def __init__(self, config):
+        remote_name = "OCPS"
+        remote_index = config.ocps_index
+        remote_info = watcher.RemoteInfo(
+            name=remote_name,
+            index=remote_index,
+            callback_names=["job_result"],
+            poll_names=[],
+        )
+        super().__init__(
+            config=config,
+            name=f"CpVerifyAlarm.{remote_info.name}:{remote_info.index}",
+            remote_info_list=[remote_info],
+        )
+
+        self.calibration_type = config.calibration_type
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            description: Configuration for CpVerifyBiasAlarm
+            type: object
+            properties:
+                calibration_type:
+                    description: >-
+                        Alarm calibration type: BIAS, DARK, or FLAT.
+                    type: string
+                ocps_index:
+                    OCPS index (e.g., OCPS:1: LATISS; OCPS:2: LSSTComCam).
+                verification_threshold:
+                    type: integer
+                    descriptor: Maximum number of failures per detector per test type.
+                default: 8
+            required: [name]
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def __call__(self, topic_callback):
+        msg = topic_callback.get()
+        # OCPS response after calling the pipetask
+        response = json.loads(msg.result)
+        # boolean: did verification fail?
+        verify_tests_pass = await self.get_verify_tests_pass_boolean(response)
+
+        if verify_tests_pass:
+            return watcher.NoneNoReason
+        else:
+            # Tests did not pass. Follow up on images and calibrations is
+            # recommended.
+            return AlarmSeverity.SERIOUS, "FAULT state"
+
+    async def get_verify_tests_pass_boolean(self, response_verify):
+        """Determine if cp_verify bias tests passed from OCPS response.
+
+        Parameters
+        ----------
+        response : `dict`
+            OCPS call response.
+
+        Returns
+        -------
+        verify_pass : `bool`
+            Did the cp_verify tests pass?
+        """
+        verify_pass = True
+        # Loop over the entries of the 'results' list
+        # and look for the adecuate dataset type.
+        for entry in response_verify["results"]:
+            if "verifyBiasStats" in entry["uri"]:
+                verify_stats_string = "verifyBiasStats"
+                break
+            elif "verifyDarkStats" in entry["uri"]:
+                verify_stats_string = "verifyDarkStats"
+                break
+            elif "verifyFlatStats" in entry["uri"]:
+                verify_stats_string = "verifyFlatStats"
+                break
+            else:
```

Comment:
How many entries are there in `results` that you need to process? This `else` statement will cause the loop to break in the first pass. If there are more than one entries you need to process you probably need to think about a better way to do this.